File: youlexuan-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService', function ($http) {

    this.save = function (entity) {
        return $http.post("../brand/addBrand.do", entity)
    }
    /*查询一个id*/
    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id=' + id)
    }
    this.batchdelete = function (selectIds) {
        return $http.get('../brand/batchdelete.do?ids=' + selectIds)
    }

//条件查询
    this.search = function (page, rows, searchEntity) {
        return $http.post('../brand/search.do?page=' + page + "&rows=" + rows, searchEntity)
    }
/*查询所有品牌得到的List<map>*/
    this.selectOptionList = function () {
        return $http.get('../brand/selectOptionList.do');
    }


})

File: youlexuan-service-interface/src/main/java/com/youlexuan/sellergoods/service/BrandService.java
package com.youlexuan.sellergoods.service;

import com.ujiuye.entity.PageResult;
import com.ujiuye.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    /*查询所有的品牌管理信息*/
    public List<TbBrand> findAll();

    /*分页查询品牌信息*/
    public PageResult findPage(TbBrand brand,int pageNum, int pageSize);

    boolean addBrand(TbBrand brand);

    TbBrand findOne(Long id);

    boolean batchdelete(Long[] ids);

    List<Map> selectOptionList();
}
